.pipelines/cni/pipeline.yaml

```diff
@@ -12,27 +12,204 @@ stages:
         steps:
           - script: |
               echo "Setting up environment"
-              echo "##vso[task.setvariable variable=npmVersion;isOutput=true]$(make npm-version)"
               go version
+              echo "##vso[task.setvariable variable=commitID;isOutput=true]$(make revision)"
+              echo "##vso[task.setvariable variable=npmVersion;isOutput=true]$(make npm-version)"
+              echo "##vso[task.setvariable variable=dropgzVersion;isOutput=true]$(make cni-dropgz-version)"
+              echo "##vso[task.setvariable variable=cnsVersion;isOutput=true]$(make cns-version)"
             name: "SetEnvVars"
             displayName: "Set Environment Variables"
             condition: always()
+
+  - stage: build_images
+    dependsOn: setup
+    displayName: "Build Images"
+    jobs:
+      - job: containerize_amd64
+        displayName: Build Images
+        pool:
+          name: "$(BUILD_POOL_NAME_LINUX_AMD64)"
+        strategy:
+          matrix:
+            cni_dropgz_test_linux_amd64:
+              arch: amd64
+              name: cni-dropgz-test
+              os: linux
+            cni_dropgz_test_windows2022_amd64:
+              arch: amd64
+              name: cni-dropgz-test
+              os: windows
+              os_version: ltsc2022
+            cns_linux_amd64:

```

Comment:
I 100% agree that we need to test CNI with cns/latest current code base. But IMO that should be tested in PR pipeline rather than CNI release pipeline. This will run once every release or so and should be responsible for testing CNI binaries with the released version of CNS/NPM